### PR TITLE
Place Page Choropleth Fixes

### PR DESCRIPTION
**Pull request by chejennifer**
- for date range in title, use "[start date] to [end date]" instead of "[start date] - [end date]" (eg. "2020-03 to 2020-04" vs "2020-03-2020-04")
- the function for getting tooltip html for place pages had a bug causing it to always show "data missing"

### CODE REVIEW

**Review by shifucun — APPROVED** [review]

File: server/routes/api/chart.py [hunk 1]
```diff
@@ -262,7 +262,7 @@ def get_date_range(dates):
     sorted_dates_list = sorted(list(dates))
     date_range = sorted_dates_list[0]
     if len(sorted_dates_list) > 1:
-        date_range = f'{sorted_dates_list[0]} - {sorted_dates_list[-1]}'
+        date_range = f'{sorted_dates_list[0]} to {sorted_dates_list[-1]}'
```
Comment (beets) [inline server/routes/api/chart.py:265]: is this change strictly necessary? this would require translations for place pages.

how about the [endash](https://www.thepunctuationguide.com/en-dash.html) instead?

**Review by beets — APPROVED** [review]

File: server/routes/api/chart.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/api/chart.py:265]: This isn't necessary but it just might look a little confusing if there is a date range like "2020-03-2020-04". However, there usually aren't date ranges on the place pages so I think I'll just use the endash instead.
